backend/Tasque.WebAPI/Tasque.Core.WebAPI.csproj

```diff
@@ -12,7 +12,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Tasque.BLL\Tasque.Core.BLL.csproj" />
+    <ProjectReference Include="..\Tasque.DAL\Tasque.Core.DAL.csproj" />
```

Comment:
two ItemGroup-s, when expected one

Reply:
Yes, when I corrected the conflicts, I deleted them. Already fixed it